index.md

```diff
@@ -195,96 +129,113 @@ either the result of a digital migration or information that enables the creatio
+### Generation
 
-## Cardinality of the SIP to AIP transformation
+If the logical AIP is changed, the physical representation of the information 
+in a container may change as well. We call a *generation* of the AIP a new 
+manifestation of physical container files representing a new version of a 
+logical AIP. The new generation of the AIP can be a single container file or a 
+set of container files.
 
-Regarding the transformation of SIPs to AIPs, in the OAIS it is stated that "one
-or more SIPs are transformed into one or more Archival Information Packages (AIPs)
-for preservation [@OAIS2012, p. 2-8]”.
+A generation of the AIP is created as a consequence of preservation policy 
+decisions. These are generally applied across AIPs without reference to their 
+source or the content contained in them. Examples:
 
-The recommendation in this regard is to guarantee a one-to-one relationship
-between the logical SIP and the logical AIP whereas the relation of the
-corresponding physical packages of SIPs and AIPs can be any kind of many-to-many
-mapping.
+- Changes to the information package structure to conform to a new specification 
+- Replacing all PDF v1.4 -> 2.0s with PDF/A equivalents due to archival policy change
 
-# AIP format specification
+### Version
 
-The following AIP format specification is defined as a set of requirements[^2]
-which will be explained with the help of textual descriptions, figures, and
-concrete examples. It is divided into two parts. On the one hand, there is the
-structure and metadata specification which defines how the AIP is conceptually
-organized by means of a folder hierarchy in a file system and a set of
-metadata standards. And on the other hand, there is the physical container
-specification which defines the bit-level manifestation of the transferable
-entity.
+A version of an AIPs is created due to some change in the content or metadata, or 
+a policy for a particular collection:
 
-[^2]: The requirements terminology is based upon RFC2119, "Key words for use in
-      RFCs to indicate requirement levels", RFC 2119, S. Bradner, March 1997.
-      Available at: http://www.ietf.org/rfc/rfc2119.txt
+- Extending, correcting or removing package metadata
+- Adding, replacing or removing content items 
 
-<a name="csip"></a>
 
-## Overview about the CSIP
+## Segmentation of the AIP <a name="structdiv"></a>
 
-In the following, we will briefly describe the structure of an IP as defined
-by the CSIP. Against this background, the AIP format will be introduced as a
-container format which allows managing the life-cycle of an E-ARK IP that
-starts with the ingest of an SIP.
+From the point of view of preserving the integrity of the AIP, the ideal case 
+isthat the logical AIP is packaged as one single physical container, because 
+all themetadata and content required to interprete the information package is 
+available inone single entity. In reality, however, this is not always possible 
```

Comment:
available inone single entity. - available in a single entity.

Single is one by definition ;)

Reply:
done